# Description: launchpad integration changes
# UbuntuSpecific: ubuntu and launchpad
#
Index: evince-2.32.0/configure.ac
===================================================================
--- evince-2.32.0.orig/configure.ac	2010-09-28 04:48:44.000000000 +1000
+++ evince-2.32.0/configure.ac	2010-12-06 12:53:23.388902002 +1100
@@ -193,7 +193,7 @@
         ;;
 esac
 
-PKG_CHECK_MODULES([SHELL_CORE],[libxml-2.0 >= $LIBXML_REQUIRED gtk+-$GTK_API_VERSION >= $GTK_REQUIRED gio-2.0 >= $GLIB_REQUIRED gthread-2.0 $SHELL_PLATFORM_PKGS])
+PKG_CHECK_MODULES([SHELL_CORE],[libxml-2.0 >= $LIBXML_REQUIRED gtk+-$GTK_API_VERSION >= $GTK_REQUIRED gio-2.0 >= $GLIB_REQUIRED gthread-2.0 $SHELL_PLATFORM_PKGS launchpad-integration])
 
 # *********
 # SM client
Index: evince-2.32.0/data/evince-ui.xml
===================================================================
--- evince-2.32.0.orig/data/evince-ui.xml	2010-09-28 02:54:34.000000000 +1000
+++ evince-2.32.0/data/evince-ui.xml	2010-12-06 12:53:23.388902002 +1100
@@ -61,6 +61,7 @@
 
     <menu name="HelpMenu" action="Help">
       <menuitem name="HelpContentsMenu" action="HelpContents"/>
+      <placeholder name="LaunchpadItems" />
       <menuitem name="HelpAboutMenu" action="HelpAbout"/>
     </menu>
   </menubar>
Index: evince-2.32.0/shell/ev-window.c
===================================================================
--- evince-2.32.0.orig/shell/ev-window.c	2010-09-28 02:54:36.000000000 +1000
+++ evince-2.32.0/shell/ev-window.c	2010-12-06 12:53:23.388902002 +1100
@@ -30,6 +30,8 @@
 #include "config.h"
 #endif
 
+#include <launchpad-integration.h>
+
 #include <errno.h>
 #include <stdlib.h>
 #include <string.h>
@@ -6821,6 +6823,8 @@
 	ev_window->priv->menubar =
 		 gtk_ui_manager_get_widget (ev_window->priv->ui_manager,
 					    "/MainMenu");
+	launchpad_integration_add_ui (ev_window->priv->ui_manager,
+				      "/MainMenu/HelpMenu/LaunchpadItems");
 	gtk_box_pack_start (GTK_BOX (ev_window->priv->main_box),
 			    ev_window->priv->menubar,
 			    FALSE, FALSE, 0);
