diff -urN evince-2.32.0/libview/Makefile.am evince-2.32.0.new/libview/Makefile.am
--- evince-2.32.0/libview/Makefile.am	2010-09-28 02:54:34.000000000 +1000
+++ evince-2.32.0.new/libview/Makefile.am	2011-01-25 13:19:15.774612000 +1100
@@ -122,6 +122,7 @@
 	$(AM_V_GEN) PKG_CONFIG_PATH=$(top_builddir):$$PKG_CONFIG_PATH \
 	$(G_IR_SCANNER) -v --namespace EvinceView \
 	--add-include-path=$(top_builddir)/libdocument \
+	--library-path=$(top_builddir)/libdocument \
 	--strip-prefix=Ev \
 	--nsversion=$(EV_API_VERSION) \
 	--include=GLib-2.0 \
